internal/usage/usage.go

```diff
@@ -81,6 +81,11 @@ const (
 	RSName                          = "The replica set that the index is built on."
 	Key                             = "Index keys. Should be formatted as field:type."
 	Unique                          = "Create a unique key index."
+	ResourceName                    = "Name of the resource from which to collect logs."
+	ResourceType                    = "Type of resource from which to collect logs."
```

Comment:
if you decide to keep these flags let's make this more specific as the content includes logs
```suggestion
	LogResourceName                    = "Name of the resource from which to collect logs."
	LogResourceType                    = "Type of resource from which to collect logs."
```